### DISCUSSION TITLE
Bitcoin, Blockchain, NFT, Mainstream & Como evitar estafas

### DISCUSSION THREAD

**Discussion by David-Lazaro-Fernandez** [gittogethers/octogatosconf22#80] [Talks in Spanish]
***Ponente: Dulce Villareal***
> Hablaremos sobre los pilares básicos para entender la tecnología de Bitcoin & Blockchain, lo anterior nos permitirá entender Tokenomics vs Ponziconomics. Te enseñaré qué debes de saber antes de aceptar una oferta en crypto o antes de empezar a crear un producto para la industria.

**Comment by xsrpm** [comment]
Me gustaron las slides de apoyar a open source, no entendí de que iba la parte de FOSS good vibes

**Reply by dulcedu** [reply]
Se refiere a proyectos FOSS donde toda la gente y el equipo es muy buena vibra. Sabes gente super amable y humilde

**Comment by alvarengav** [comment]
Recursos para empezar?

**Reply by elchamojavier** [reply]
Yo empece siguiendo este canal: 
https://www.youtube.com/watch?v=y8YyZELnVaw

**Comment by raulgonzalezcz** [comment]
Gracias por la platica. Pueden adjuntar los links a recursos?

**Reply by dulcedu** [reply]
Si claro. 
 -  Escuela de Bitcoin y Lightning https://libreriadesatoshi.com/
 - Buscar los venture capitalist de una startup https://www.crunchbase.com/
 - Valor del mercado de un token  https://coinmarketcap.com/
 - Ver Total Value Locked All Chains  https://defillama.com/chains
 FOSS projets with amazing team 
 - https://github.com/ElementsProject/lightning/labels/good%20first%20issue 
-  github.com/lnp2pbot
GRANTS
- Bitcoin Grants https://bitcoinwords.github.io/grants/
- Crypto Grants https://identityreview.com/15-crypto-grants-you-should-know-about/

**Comment by NoemiFN** [comment]
Qué tan dificil consideran que es el mundo del blockchain? Recursos para aprender de cero?

**Reply by dulcedu** [reply]
nada es díficil, tu eres capaz de masterizar cualquier habilidad son horas de inversión.

**Reply by NoemiFN** [reply]
Voy a buscar, me imagino que debe haber cursos y libros del tema. Por algún lado empezaré! Muchas gracias!!

**Reply by dulcedu** [reply]
@NoemiFN la pregunta sería que prefieres ver videos, leer libros, blog post y que profesión tienes para darte algo que sea personalizado. 
Sabes programar?

**Reply by NoemiFN** [reply]
Prefiero ver videos, soy bastante visual. Por ahora solo soy analista de datos y sé programar en python y un poquito en javascript. Manejo un poquito de Rstudio y bastante de SQL :D

**Reply by dulcedu** [reply]
Ingles - https://www.youtube.com/playlist?list=PLPQwGV1aLnTuN6kdNWlElfr2tzigB9Nnj
Español - https://www.youtube.com/channel/UCLuWP-E8Bj2IwTQEVxYfMEw

**Reply by elchamojavier** [reply]
gracias @dulcedu - los revisare tambien!

**Comment by the-lastbit** [comment]
Saludos. 

Que recomendaría para un principiante que se quiere iniciar en blockchain?
Me llamó la atención la capa para smart contracts en Bitcoin, Pensé que solo era posible en Ethereum. Dada la situación del merge en Ethereum, cree que sea buena idea iniciar con dicha tecnología o es preferible Bitcoin?

Gracias por la charla.

**Reply by dulcedu** [reply]
Si es algo muy nuevo que nadie conoce. Pero si será posible https://www.reddit.com/r/lightningnetwork/comments/qebx92/can_you_write_smart_contracts_on_lightning_network/

**Reply by dulcedu** [reply]
https://river.com/learn/what-are-bitcoin-smart-contracts/

**Reply by the-lastbit** [reply]
Gracias por las fuentes, ahora a dedicarle tiempo 📚

**Reply by dulcedu** [reply]
También puedes tomar los seminarios de https://libreriadesatoshi.com/

**Reply by the-lastbit** [reply]
Lo checaré 😌
